pipelines/_ingredients/VerifyBias.yaml

```diff
@@ -61,3 +66,18 @@ tasks:
       hasInputResults: true
       hasMatrixCatalog: true
   analyzeBiasCore: lsst.analysis.tools.tasks.VerifyCalibAnalysisTask
+  mosaicExp8:
+    class: lsst.pipe.tasks.visualizeVisit.VisualizeMosaicExpTask
+    config:
+      connections.inputExps: 'verifyBiasProcBin8'
+      connections.outputData: 'verifyBiasResidual8'
```

Comment:
It's at times like this that I realize that the github font is terrible and should distinguish lower case `l` and `1` better.